platform/viewer/src/loadDynamicConfig.js

```diff
@@ -0,0 +1,31 @@
+export default async config => {
+  const useDynamicConfig = config.dangerouslyUseDynamicConfig;
+
+  // Check if dangerouslyUseDynamicConfig enabled
+  if (useDynamicConfig.enabled) {
+    // If enabled then get configUrl param
+    let query = new URLSearchParams(window.location.search);
+    let configUrl = query.get('configUrl');
+
+    if (!configUrl) {
+      // Handle OIDC redirects
+      const obj = JSON.parse(sessionStorage.getItem('ohif-redirect-to'));
+      if (obj) {
+        const query = new URLSearchParams(obj.search);
+        configUrl = query.get('configUrl');
```

Comment:
You never do anything with the configUrl beyond this point, so there is no point assigning anything here.

Reply:
good catch